### PR TITLE
[Cleanup][A-9] clean some VarType for test 

### PR DESCRIPTION
**Pull request by co63oc**
### PR types

Others 
### PR changes

Others 
### Description

https://github.com/PaddlePaddle/Paddle/issues/61385

### CODE REVIEW

File: python/paddle/base/dygraph/math_op_patch.py
Comment (SigureMo) [inline python/paddle/base/dygraph/math_op_patch.py:1]: 这个文件先不要改吧

Comment (co63oc) [reply python/paddle/base/dygraph/math_op_patch.py:1]: 已修改

**Comment by luotao1** [comment]
@SigureMo 可以再review一下

**Review by SigureMo — APPROVED** [review]
LGTMeow

**Pull request merged**